source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,75 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.6 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: March 15, 2024**
+
+### Compatibility
+ - Updated minimum required Edge and Chrome versions to 120+.
+
+```{Important}
+If you upgrade from a release earlier than v9.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls version [v0.24.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.24.0).
+ - Pre-packaged GitLab plugin version [v1.8.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0).
+ - Re-designed the **System Console -> User Management** screen, and added a batch CSV export of users.
+ - Changed notifications so that, if you have **Thread Reply Notifications** disabled for your account and "Automatically follow threads in this channel" for a channel, you'll no longer receive thread notifications for that channel.
```

Comment:
```suggestion
 - Changed notifications so that, if you have **Thread Reply Notifications** disabled for your account and **Automatically follow threads in this channel** for a channel, you'll no longer receive thread notifications for that channel.
```

Reply:
@harshilsharma63 @devinbinnie Would you know the answer to this? (For https://github.com/mattermost/mattermost/pull/26062)

Reply:
@cwarnermm @amyblais It should be enabled.